docs/subscription-plans.rst

```diff
@@ -12,15 +12,26 @@ customer is spared the complexity of finding the exact required hardware
 combinations themselves.
 
 At the same time, the plans also offer flexibility, since your use case may
-change. Not only is it possible to switch plans, but within a given plan you
-can scale the required capacity for a cluster up or down. In order to make it
-transparent and yet easy to use, this scaling is measured in terms of Database
-Transaction Units (DTUs).
+change. Not only do we offer multiple plans with different specifications, but
+within a given plan you can scale the required capacity for a cluster up or
+down in two ways: either by adding or subtracting nodes or by changing tier
+within a given plan.
+
+In order to make the costs for a given set of hardware transparent and yet easy
+to use, this scaling is measured in terms of Database Transaction Units (DTUs).
+The simple rule of thumb is that plans and tiers increase the price per DTU,
+whereas the number of nodes represents the number of DTUs expended for an hour
+of usage.
 
 This reference gives a brief description of the plans and subsequently explains
-the meaning and usage of DTUs for scaling and pricing within each plan. This
-should make it easy to choose the right plan for your CrateDB Cloud
-subscription.
+the meaning and usage of nodes, tiers, and DTUs. This should make it easy to
+choose the right setup for your CrateDB Cloud subscription.
+
+.. NOTE::
+    `CrateDB Edge`_ works differently. Since it allows hosting CrateDB Cloud
```

Comment:
Technically there is a plan for edge, but the price is 0.

Reply:
Agreed ;) But we might want to specify that edge is free for now, but will not always be?

Reply:
Fair point :)